.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "workbench.colorCustomizations": {}
+}
```

Comment:
There should be an option to set your editor to automatically trim whitespace and add a newline to the end of the file on save